library/std/src/sys/pal/uefi/process.rs

```diff
@@ -502,6 +510,10 @@ mod uefi_command_internal {
                     ((*bt.as_ptr()).unload_image)(self.handle.as_ptr());
                 }
             }
+
+            if let Some((ptr, len)) = self.args {
+                let _ = unsafe { Box::from_raw(crate::slice::from_raw_parts_mut(ptr, len)) };
```

Comment:
I *think* this is correct, but I'd still prefer not to create a reference here:
```suggestion
                let _ = unsafe { Box::from_raw(ptr::slice_from_raw_parts_mut(ptr, len)) };
```